src/scripts/candidate_calc_outside_spending.js

```diff
@@ -0,0 +1,197 @@
+
+const fs = require('fs');
+const fetch = require('node-fetch');
+const parse = require('csv-parse');
+const parseSync = require('csv-parse/lib/sync');
+
+const candidate_information_url = "https://docs.google.com/spreadsheets/d/1mENueYg0PhXE_MA9AypWWBJvBLdY03b8H_N_aIW-Ohw/export?format=csv&gid=0";
+const assets_path = '../assets/data/';
+const candidates_path = '../assets/candidates/';
+const election_year = '2020';
+const netFile_API_csv_filenames = ['netfile_api_2019.csv', 'netfile_api_2020.csv'];
+
+
+/**
+ * Reads from a Google Sheet and returns a list of candidate names in an array of strings.
+ * @returns {Promise <string[]>}
+ */
+async function getCandidateNames() {
+  const SHEET_URL = candidate_information_url;
+  let response, fileData;
+
+  try {
+    response = await fetch(SHEET_URL);
```

Comment:
hmm, maybe just use candidate_information_url instead of SHEET_URL, or rename it on line 7, so there's no need for line 19?